lib/guard/teaspoon/runner.rb

```diff
@@ -18,14 +18,14 @@ def initialize(options = {})
       end
 
       def run_all(options = {})
-        # run all tests instead of running only the last run tests
-        @options.delete :files
         @console.execute(@options.merge(options))
       end
 
       def run(files = [], options = {})
         return false if files.empty?
-        @console.execute(@options.merge(options).merge(files: files))
+        run_options = @options.merge(options).merge(files: files)
+        Compat::UI.info "Running files: " + files.join(', ') if run_options[:show_filenames]
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.